src/Raven.Client/Documents/Operations/TimeSeries/GetTimeSeriesOperation.cs

```diff
@@ -13,44 +13,41 @@ public class GetTimeSeriesOperation : IOperation<TimeSeriesDetails>
     {
         private readonly string _docId;
         private readonly string _timeseries;
-        private readonly DateTime _from, _to;
+        private readonly List<(DateTime, DateTime)> _ranges;
```

Comment:
I think we need to introduce a class e.g. `TimeSeriesRange`. By definition classes are more extendable or easier to modify than tuples. You can mark properties  as obsolete, add new ones, etc. All without breaking any compatibility.